app/models/presenters/plate_presenter.rb

```diff
@@ -58,8 +59,8 @@ def number_of_wells
       "#{number_of_filled_wells}/#{size}"
     end
 
-    def pcr_cycles
-      pcr_cycles_specified.zero? ? 'No pools specified' : cycles.to_sentence
+    def requested_pcr_cycles
+      pcr_cycles.empty? ? 'No pools specified' : pcr_cycles.to_sentence
```

Comment:
Is pooling related to PCR cycles? odd text

Reply:
The only scenario where we could reasonably expect no PCR cycles to be specified, is one in which there were no pools defined on the plate.